app/views/request/show.html.erb

```diff
@@ -165,6 +165,17 @@
     </div>
   <% end %>
 
+  <% unless @render_to_file %>
```

Comment:
I also didn't find styling the `Follow | 2 followers` bit straightforward in the Italy theme but I only tried it very lazily and that "problem" might be me being tired (or the theme being inconsistent, although as I've been working on it today, functionally that's the same thing)